recipes/poppler/all/conandata.yml

```diff
@@ -6,7 +6,7 @@ patches:
   "20.09.0":
     - patch_file: "patches/0001-cmake-use-PROJECT_SOURCE_DIR-fix-CMAKE_MODULE_PATH.patch"
       base_path: "source_subfolder"
-    - patch_file: "patches/0002-cmake-use_CONAN_PKG-targets.cmake"
+    - patch_file: "patches/0002-cmake-use_CONAN_PKG-targets.patch"
       base_path: "source_subfolder"
     - patch_file: "patches/0003-cmake-disable-tests.cmake"
```

Comment:
+1

Reply:
oh didn't see that one

Reply:
you did not rename the file(s) either

Reply:
In my defence, I did `0002-cmake-use_CONAN_PKG-targets.patch` :smile: